index.js

```diff
@@ -0,0 +1,161 @@
+setWallpaper("_xWallpapers");
+
+function setInStorage(obj) {
+  chrome.storage.local.set(obj);
+}
+
+function setWallpaper(key) {
+  chrome.storage.local.get(key, async (result) => {
+    const cache = result[key];
+    if (cache) {
+      const randomNum = Math.ceil(Math.random() * (79 - 0) + 0);
```

Comment:
Couldn't understand why you're using (79-0) + 0

Reply:
api gives 80 wallpapers in every request, so saving them in localstorage then using any one random on refresh